doc/crm.8.adoc

```diff
@@ -2036,7 +2036,8 @@ stop <rsc> [<rsc> ...]
 ==== `trace`
 
 Start tracing RA for the given operation. When `[<log-dir>]`
-is not specified the trace files are stored in `$HA_VARLIB/trace_ra`.
+is not specified, the trace files are stored in `$HA_VARLIB/trace_ra`.
+If $HA_VARLIB is not defined, it defaults to `/var/lib/heartbeat`.
```

Comment:
1. `HA_VARLIB` is not used in the latest patch.

    ```suggestion
    is not specified, the trace files are stored in `/var/lib/heartbeat/trace_ra`.
    ```

2. config template `etc/crm.conf` needs update:

    ```patch
    -; heartbeat_dir = <detected>
    +; heartbeat_dir = /var/lib/heartbeat
    ```